cantor-grpc-client/pom.xml

```diff
@@ -29,6 +29,12 @@
     </properties>
 
     <dependencies>
+        <!-- CANTOR PHOENIX -->
```

Comment:
The grpc-client module should not have a dependency on the phoenix implementation.